python-lib/uipath_client.py

```diff
@@ -43,6 +57,10 @@ def post(self, url, headers=None, json={}):
         if json is not None:
             args["json"] = json
         response = requests.post(url, **args)
+        if response.status_code >= 400:
+            logger.error("HTTP error {} on post to '{}'".format(response.status_code, url))
+            logger.error("reponse: {}".format(response.content))
```

Comment:
Couldn't you use response.text instead of response.content? Thus, the logging will be more consistent with the exception

Reply:
Done in 7b0e6bc